### PR TITLE
Improve guesses/stopwatch display and add button to reveal new hint

### LINKED ISSUE #169: Create a button to get new hint when blocked
When a user is blocked and does not know anything about what to guess, it would be nice to have a button to unlock a new guess instead of forcing him to type a random name.

### LINKED ISSUE #176: Improve display of guesses and timer during game
The current display is handy, but we could improve it to make it smoother. Maybe we could use a progress bar (see [this](https://ui.nuxt.com/components/progress)) for the number of guesses, and add animations on timer change (see [this](https://dribbble.com/search/timer) for inspo).

### PR DESCRIPTION
**Pull request by JEANPRBT**
This PR improves the display of the number of guesses and of the stopwatch, which allows to add a button to reveal a new hint, avoiding the hassle of guessing a random celebrity. See picture below for new display, both on desktop and mobile devices. It should be merged after PR #164 since it already merged the major changes of this branch. 
Closes #176 
Closes #169 

new_display_desktop
new_display_mobile

See background color slightly changed upon game finish. 

new_display_game_over

### CODE REVIEW

**Comment by JEANPRBT** [comment]
Following @roxannecvl's [comment](https://github.com/roxannecvl/whokipedia/pull/177#pullrequestreview-2043502405) on former PR :
> I haven't looked at the code yet but I have a few comments already. It looks prettier in general but I don't like that you don't know how many guesses you have in total and that you should approximate according to the loading bar. The "+" button is a great idea but I don't think that it is clear that it gives you a new hint and counts as a new guess. Also it shouldn't be there when the game is won I think.

@roxannecvl I added the total number of guesses on the right so that it is always visible. Also, I added the + button feature in the rules so that the player can understand what is it doing. Finally, this same button is now hidden when game is over.

**Comment by JEANPRBT** [comment]
Following @martinctl remarks on Discord, here is a video of the updated behavior. Now, help button keeps square on mobile devices. The total number of guesses is moving with the current number of guesses instead of being stuck on end of the progress bar. Also, text is now "hints" instead of "consumed hints", and there is a small vertical divider between number of hints and guesses.

https://github.com/roxannecvl/whokipedia/assets/66010389/2571ca74-cd2f-47c6-baf7-13c75501a76f

File: utilities/Utils.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ export function formatTime(seconds : number, simple : boolean = false){
         return `${hours} hr ${remainingMinutes} min ${formattedSeconds} sec`
     }
     if(simple) return `${minutes} m ${formattedSeconds} s`
-    return `${minutes} min ${formattedSeconds} sec`
+    return `${minutes} min ${formattedSeconds} s`
```
Comment (roxannecvl) [inline utilities/Utils.ts:148]: We have the simple format if you want to use that but mixing min and s is not very coherent. You could use simple and get ... m ... s

File: views/WelcomePageView.vue [hunk 2]
```diff
@@ -80,11 +80,16 @@ onMounted(() => {
...
-                  or until <span class="text-primary">all hints are consumed</span></li>
+              <ul class="list-inside m-2">
+                <li><UIcon name="i-heroicons-arrow-long-right-16-solid"/>
+                  Each unsuccessful guess brings a <span class="text-primary">new hint</span>.</li>
+                <li><UIcon name="i-heroicons-arrow-long-right-16-solid"/>
+                  Guess until you find the <span class="text-primary">celebrity</span> or until
+                  <span class="text-primary">all hints are consumed</span>.</li>
+                <li><UIcon name="i-heroicons-arrow-long-right-16-solid"/>
+                  If you do not have the slightest <span class="text-primary">idea</span>, press the
+                  <UIcon class="text-primary" name="i-heroicons-plus-circle-16-solid"/> button to get a new hint. Be
```
Comment (roxannecvl) [inline views/WelcomePageView.vue:91]: check if there is a class you can add (maybe top margin or base align) for this icon to be more aligned with the text

**Review by roxannecvl — APPROVED** [review]
Awesome job ! 
Try addressing the two comments and moving the + button before merging. Also if you could add "Scotland" : "Scottish" to our list of countries, we're currently missing it

File: views/WelcomePageView.vue (hunk 2 above)
Comment (JEANPRBT) [reply views/WelcomePageView.vue:91]: unfortunately i did not find any class which works

**Pull request merged**